src/frontend/src/settings.tsx

```diff
@@ -70,6 +85,46 @@ export const Settings = ({ invite }: { invite?: string }) => {
         setName(name);
     };
 
+    const onAvatarChange = async (ev: any) => {
+        ev.preventDefault();
+        if (!ev.target.files[0]) {
+            return;
+        }
+        const max_blob_size_bytes =
+            window.backendCache.config.max_blob_size_bytes;
+        let blob = null;
+        for (let size of [768, 512, 384, 256, 128, 64, 32]) {
```

Comment:
Is this for a better quality when downscaling?

Reply:
Yeah, and also for simplicity. I was not in the mood for the binary search :D